llvm/include/llvm-c/DebugInfo.h

```diff
@@ -1415,6 +1415,52 @@ LLVMMetadataRef LLVMInstructionGetDebugLoc(LLVMValueRef Inst);
  */
 void LLVMInstructionSetDebugLoc(LLVMValueRef Inst, LLVMMetadataRef Loc);
 
+/**
+ * Create a new descriptor for a label
+ *
+ * \param Builder         The DIBuilder.
+ * \param Scope           The scope to create the label in.
+ * \param Name            Variable name.
+ * \param NameLen         Length of variable name.
+ * \param File            The file to create the label in.
+ * \param LineNo          Line Number.
+ * \param AlwaysPreserve  Preserve the label even if the optimizer wants to remove it.
```

Comment:
I noticed after the fact that this line is longer than 80 characters and that the rest of the lines in this file are 80 characters. I would make a change like this but I also don't want to run a pipeline right now for a simple comment change. Waiting for review then, if this is desired, I can add it in. 
```suggestion
 * \param AlwaysPreserve  Preserve the label regardless of optimization.
```